static/app/views/performance/transactionSummary/tagExplorer.tsx

```diff
@@ -156,11 +184,7 @@ const renderBodyCell = (
       text = `${pct} faster`;
     }
 
-    return (
-      <Tooltip title={PerformanceDuration({milliseconds: dataRow.aggregate})}>
-        {t(text)}
-      </Tooltip>
-    );
+    return t(text);
```

Comment:
This translation won't do anything because `text` is a variable. You'll need to do something like

```
localValue > 1 ? t('+%s slower', pct) : t('%s faster', pct)
```

or some variation of it.